### PR TITLE
pair programming fun

### PR DESCRIPTION
**Pull request by 0blu3**

### CODE REVIEW

**Comment by shortaj** [comment]
thanks for the help

File: app.js [hunk 1]
```diff
@@ -1,110 +1,163 @@
...
+
+    var diet = prompt('Am I a vegan?').toUpperCase();
+    if (diet === 'YES') {
+      alert('You are right!');
+      correctAnswers++;
+      alert('Your score is now ' + correctAnswers + ' out of 7.');
+    } else if (diet === 'NO') {
+      alert('False!');
+    } else {
+      alert('Your answer didn\'t make sense to me.')
```
Comment (nhuntwalker) [inline app.js:49]: semicolon

File: app.js [hunk 2]
```diff
@@ -1,110 +1,163 @@
...
-    } else if (i === 4 && no) {
-      incorrect(userAnswer);
+    alert('I didn\'t understand your answer.');
+    };
+
+    console.log('Do you think I have a dog?');
+    console.log(pet);
+
+    var diet = prompt('Am I a vegan?').toUpperCase();
+    if (diet === 'YES') {
```
Comment (nhuntwalker) [inline app.js:42]: Answers should be "YES", "Y", "NO", or "N"

File: app.js [hunk 3]
```diff
@@ -1,110 +1,163 @@
...
+      alert('You are right!');
+      correctAnswers++;
+      alert('Your score is now ' + correctAnswers + ' out of 7.');
+    } else if (diet === 'NO') {
+      alert('False!');
+    } else {
+      alert('Your answer didn\'t make sense to me.')
+    };
+
+    console.log('Am I a vegan?')
```
Comment (nhuntwalker) [inline app.js:52]: semicolon

File: app.js [hunk 4]
```diff
@@ -1,110 +1,163 @@
...
+
+    var phone = prompt('Do you think I have an Android phone?').toUpperCase();
+    if (phone === 'NO') {
+      alert('False, I prefer Android phones.');
+    } else if (phone === 'YES') {
+      alert('Correct!');
+      correctAnswers++;
+      alert('Your score is now ' + correctAnswers + ' out of 7.');
+    } else {
+      alert('That answer didn\'t make sense.')
```
Comment (nhuntwalker) [inline app.js:91]: semicolon

**Pull request merged**